api/bots_api/bot_lib.py

```diff
@@ -51,7 +51,7 @@ def show_error_and_exit(self):
         sys.exit(1)
 
 def send_reply(message, response, email, send_message):
-    # type: (Dict[str, Any], str, str, Any) -> Dict[str, Any]
+    # type: (Dict[str, Any], str, str, Callable[[Dict[str, Any]], Dict[str, Any]]) -> Dict[str, Any]
```

Comment:
Hmmm... Is it possible to run Mypy's type inference on a code to automatically do the annotation? `autopep484`?